wordle-web/src/views/AboutView.vue

```diff
@@ -1,17 +1,20 @@
 <template>
-  <v-sheet color="blue" height="170px" class="pt-10 my-5">
-    <v-card color="indigo-darken-3" class="ma-5" elevation="5" rounded="xl">
-      <v-card-title>Hello World</v-card-title>
+  <v-sheet class="pt-10 my-5">
+    <v-card color="grey-darken-3" max-height="275" class="ma-5 pa-5" elevation="5" rounded="xl">
+      <v-card-title>About</v-card-title>
+      <body>
+        This has been an interesting class so far, but I am not looking forward to the c# section as
+        I have yet to use the language and this class is already taking up a decent amount of time.
+        In terms of the current project, I'm struggling a lot with basic formatting with my current
+        headache being my columns on the keyboard not having any spacing between them on a small
```

Comment:
Getting the mobile styling right is difficult. You won't lose any points for it not being mobile friendly, so don't stress about it